\doxysection{Référence de la classe django.\+core.\+exceptions.\+Multiple\+Objects\+Returned}
\hypertarget{classdjango_1_1core_1_1exceptions_1_1MultipleObjectsReturned}{}\label{classdjango_1_1core_1_1exceptions_1_1MultipleObjectsReturned}\index{django.core.exceptions.MultipleObjectsReturned@{django.core.exceptions.MultipleObjectsReturned}}


Graphe d\textquotesingle{}héritage de django.\+core.\+exceptions.\+Multiple\+Objects\+Returned\+:
% FIG 0


Graphe de collaboration de django.\+core.\+exceptions.\+Multiple\+Objects\+Returned\+:
% FIG 1


\doxysubsection{Description détaillée}
\begin{DoxyVerb}The query returned multiple objects when only one was expected.\end{DoxyVerb}
 

La documentation de cette classe a été générée à partir du fichier suivant \+:\begin{DoxyCompactItemize}
\item 
env/lib/python3.\+12/site-\/packages/django/core/\mbox{\hyperlink{django_2core_2exceptions_8py}{exceptions.\+py}}\end{DoxyCompactItemize}
